pkg/image/utilities/utilities.go

```diff
@@ -99,32 +100,43 @@ func GatherTargetImagesAndSources(systemChartsPath, chartsPath string, imagesFro
 		return ImageTargetsAndSources{}, fmt.Errorf("%s: %w", "could not write rancher-rke-k8s-versions.txt file", err)
 	}
 
-	externalImages := make(map[string][]string)
-	k3sUpgradeImages, err := ext.GetExternalImages(rancherVersion, data.K3S, ext.K3S, &semver.Version{
+	k8sVersion1_21_0 := &semver.Version{
 		Major: 1,
 		Minor: 21,
 		Patch: 0,
-	})
+	}
+
+	externalLinuxImages := make(map[string][]string)
+	externalWindowsImages := make(map[string][]string)
+
+	k3sUpgradeImages, err := ext.GetExternalImages(rancherVersion, data.K3S, ext.K3S, k8sVersion1_21_0, img.Linux /* unused for k3s */)
```

Comment:
I'm a bit confused on what the comment here is indicating, what is unused?

Reply:
Removed the comment as it was not necessary & confusing